contracts/voting/dao-voting-cw4/src/contract.rs

```diff
@@ -26,6 +29,18 @@ pub fn instantiate(
 ) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
 
+    let config: Config = if let Some(active_threshold) = msg.active_threshold {
+        Config {
+            active_threshold: Some(active_threshold),
+        }
+    } else {
+        Config {
+            active_threshold: None,
+        }
+    };
+
+    CONFIG.save(deps.storage, &config)?;
```

Comment:
This config struct is really not necessary. You forget to save `ACTIVE_THRESHOLD` here.

Reply:
Don't forget to validate that it's not a percent.